src/database/queries/getData.js

```diff
@@ -24,6 +24,12 @@ const getSpecalize = (categoryId) => {
   };
   return connect.query(sql);
 };
+const getProposals = (specId) => {
+  const sql = 'select proposal.title,  proposal.description, proposal.id, users.firstname,users.lastname,users.photo_url  from proposal inner join users on proposal.user_id = users.id where proposal.specalization_id = $1';
+  const values = [specId];
+  return connect.query(sql, values);
```

Comment:
@AhmedAl-Almi This looks good! My only suggestion is to add spaces after the user fields in the query: `... users.firstname, users.lastname, users.photo_url from proposal ...` to make it a little easier to read.

Reply:
Sure, thanks Tom